### ISSUE TITLE
[ Recomendation ] Github wiki

### ISSUE THREAD

**Issue by robertlc04** [nicbarker/clay#60]
Maybe can be better use(create) a wiki for all the main documentation. This may help for read and search the specific data you want to use of you have doubt.

**Comment by nicbarker** [comment]
I agree, the README started as a single document but then grew into a monster. It would definitely make sense to migrate it to something that had nice collapsible sections and easier searching.

**Comment by robertlc04** [comment]
You can enable the repo wiki? I made a very simple wiki can be an start
